package.json

```diff
@@ -0,0 +1,38 @@
+{
```

Comment:
Not related to this file, but there is no need for the .DS_Store file to be included here